client-app/shared/checkout/components/select-address-modal.vue

```diff
@@ -239,8 +249,18 @@ const columns = computed<ITableColumn[]>(() =>
         { id: "phone", title: t("common.labels.phone") },
         { id: "email", title: t("common.labels.email") },
         { id: "id", title: t("common.labels.active_address"), align: "center" },
-      ],
-);
+      ];
+
+  if (hasFavoriteAddresses.value) {
+    return [{ id: "isFavorite", classes: "w-12" } as ITableColumn].concat(cols);
+  }
+
+  return cols;
+});
+
+function isFavoriteAddress(address: AnyAddressType): boolean {
+  return "isFavorite" in address && address.isFavorite;
```

Comment:
return address?.isFavorite

Reply:
Nope. Some types of `AnyAddressType` could not include the property `isFavotite` (i.e. `OrderAddressType`).

Reply:
According to your code this property may be there. So you need to include it in the type